lib/cheri-libc++/__config_site

```diff
@@ -0,0 +1,42 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _LIBCPP_CONFIG_SITE
+#define _LIBCPP_CONFIG_SITE
+
+/* #undef _LIBCPP_ABI_VERSION */
+/* #undef _LIBCPP_ABI_UNSTABLE */
```

Comment:
I do wonder if we should use the new libc++ ABI at some point, although we probably want it for the baseline as well if we want more comparable benchmarks.

Reply:
I think upstream FreeBSD (maybe LLVM?) is looking at moving to ABI version 2.  FreeBSD has some local hack to avoid an ABI break that is a source of friction within LLVM IIUC.